### DISCUSSION TITLE
v2.16.1

### DISCUSSION THREAD

**Discussion by JorTurFer** [kedacore/keda#6447] [Announcements]
We are happy to release KEDA v2.16.1 🎉

Here are some fixes:

- **General**: Centralize and improve automaxprocs configuration with proper structured logging ([#5970](https://github.com/kedacore/keda/issues/5970))
- **General**: Paused ScaledObject count is reported correctly after operator restart ([#6321](https://github.com/kedacore/keda/issues/6321))
- **General**: ScaledJobs ready status set to true when recoverred problem ([#6329](https://github.com/kedacore/keda/pull/6329))
- **Selenium Grid Scaler**: Exposes sum of pending and ongoing sessions to KDEA ([#6368](https://github.com/kedacore/keda/pull/6368))
- **Selenium Grid Scaler**: Selenium Grid in case multiple scaler triggers are activate ([#6437](https://github.com/kedacore/keda/pull/6437))

Learn how to deploy KEDA by reading [our documentation](https://keda.sh/docs/2.16/deploy/).

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/keda/releases/tag/v2.16.1'>v2.16.1</a>.</em>

**Comment by ibraheemcisse** [comment]
Hello @JorTurFer , I wanted to please ask for your help and other contributors. I'm encountering the following error when trying to apply the KEDA CRDs:

`The CustomResourceDefinition "scaledjobs.keda.sh" is invalid: metadata.annotations: Too long: must have at most 262144 bytes`

However, I ran into the issue where the metadata.annotations field is too long for the **scaledjobs.keda.sh CRD.** and "**triggerauthentication**" when I tried to install them sepearately.

Steps Taken:

- Attempted to apply the CRDs from the provided GitHub URL directly.

- I tried using a python script to modify the yam files to only include the necessary aspects.

Has anyone encountered this issue? How can I resolve it, or is there a recommended way to install the CRDs without hitting the metadata annotations size limit?

Thank you.

**Reply by adit-bala** [reply]
Hello, this topic has been discussed previously, and here are some potential solutions:  
- **[GitHub Issue #5245](https://github.com/kedacore/keda/issues/5245)** – The recommended approach is to use **server-side apply** to resolve the issue.  
- **[GitHub Issue #6441](https://github.com/kedacore/keda/issues/6441)** – If you prefer to avoid server-side apply, this thread suggests an alternative, albeit a more unconventional workaround.

**Reply by ibraheemcisse** [reply]
Hello @adit-bala , Thank you for sharing. This approach handled the large CRDs perfectly.
`kubectl apply --server-side -f https://github.com/kedacore/keda/releases/download/v2.12.1/keda-2.12.1-core.yaml  `
Best Regards, 
Ibrahim.
